src/components/share-structure/ShareStructure.vue

```diff
@@ -320,7 +320,7 @@
             <td class="text-right">
               {{ row.item.parValue ? `$${formatParValue(row.item.parValue)}` : 'No Par Value' }}
             </td>
-            <td>{{ row.item.currency }}</td>
+            <td>{{ row.item.parValue ? row.item.currency : null }}</td>
```

Comment:
Please show a screenshot of this.

Otherwise, lgtm and I will publish it once the PR is merged.